scripts/lib/CIME/XML/component.py

```diff
@@ -282,3 +282,36 @@ def print_values(self):
             name = self.get(entry, "id")
             text = self.text(self.get_child("desc", root=entry))
             logger.info("   {:20s} : {}".format(name, text.encode('utf-8')))
+
+    def return_values(self):
```

Comment:
Hey @jedwards4b , i think I remember deleting these "return_values" methods because they were not being used. Was I wrong? git grep confirms nothing in cime is using them.

Reply:
I think that they are used by some web interface code outside of cime: @bertinia?

Reply:
OK, we should add a comment somewhere since checkers and manual inspection will flag these methods as dead code.